sonic-xcvrd/xcvrd/xcvrd.py

```diff
@@ -508,7 +508,7 @@ def post_port_sfp_dom_info_to_db(is_warm_start, port_mapping, stop_event=threadi
         if asic_index is None:
             helper_logger.log_warning("Got invalid asic index for {}, ignored".format(logical_port_name))
             continue
-        rc = post_port_sfp_info_to_db(logical_port_name, port_mapping, xcvr_table_helper.get_int_tbl(asic_index), transceiver_dict, stop_event)
+        rc = post_port_sfp_info_to_db(logical_port_name, port_mapping, xcvr_table_helper.get_intf_tbl(asic_index), transceiver_dict, stop_event)
```

Comment:
Hi @Junchao-Mellanox Do you know why we were not able to catch this earlier ? Does mellanox platform not run the post_port_sfp_dom_info_to_db for some reason ?

Reply:
I will check why unit test cannot cover such case.